pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/DetachedTestCase.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<test-data
+    xmlns="http://pmd.sourceforge.net/rule-tests"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/rule-tests http://pmd.sourceforge.net/rule-tests_1_0_0.xsd">
+    <test-code>
+        <description>correct test case annotation</description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+public class MyTest {
+  @Test
+  public void someTest() {}
+
+  @Test
+  public void someOtherTest() {}
+}
+     ]]></code>
+    </test-code>
+    <test-code>
+        <description>missing annotation on public test case</description>
+        <expected-problems>1</expected-problems>
```

Comment:
Forgot to mention: there is also the element `expected-linenumbers` which we should use here to make sure, the rule violation detects the correct method:

```
<expected-linenumbers>5</expected-linenumbers>
```

Reply:
Good point, I'll add them too!